homeassistant/components/yalexs_ble/lock.py

```diff
@@ -0,0 +1,63 @@
+"""Support for Yale Access Bluetooth locks."""
+from __future__ import annotations
+
+from typing import Any
+
+from yalexs_ble import ConnectionInfo, LockInfo, LockState, LockStatus
+
+from homeassistant.components.lock import LockEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import YALEXSBLEEntity
+from .models import YaleXSBLEData
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up locks."""
+    data: YaleXSBLEData = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities([YaleXSBLELock(data)])
+
+
+class YaleXSBLELock(YALEXSBLEEntity, LockEntity):
+    """A yale xs ble lock."""
+
+    _attr_has_entity_name = True
+    _attr_name = "Lock"
```

Comment:
Shouldn't we set this to `None` so the entity is named after the device?

Reply:
Yes, I think that makes more sense.  There is only one lock entity

Will adjust

Reply:
e50cdd3867ee3303f69d3ad31e14c5f5bd699d78